packages/react-native/React/Base/RCTTouchHandler.m

```diff
@@ -128,8 +128,8 @@ - (void)_recordNewTouches:(NSSet *)touches
 #else // [macOS
     // -[NSView hitTest:] takes coordinates in a view's superview coordinate system.
     // The assumption here is that a RCTUIView/RCTSurfaceView will always have a superview.
-    CGPoint touchLocation = [self.view.superview convertPoint:touch.locationInWindow fromView:nil];
-    NSView *targetView = [self.view hitTest:touchLocation];
+    const CGPoint touchLocationInSuper = [self.view.superview convertPoint:touch.locationInWindow fromView:nil];
+    NSView *targetView = [self.view hitTest:touchLocationInSuper];
```

Comment:
Purely a style / preference to avoid some dot notation and use a full name
```suggestion
    NSView *superview = [[self view] superview];
    const CGPoint touchLocationInSuperview = [superview convertPoint:touch.locationInWindow fromView:nil];
    NSView *targetView = [self.view hitTest:touchLocationInSuperview];
```